bazel/api-golden/test/goldens/test_package/index.md

```diff
@@ -0,0 +1,12 @@
+## API Report File for "test-package"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export const fromTopLevel = true;
```

Comment:
Shouldn't these be types?

Reply:
It is, for a constant the actual literal value is the concrete type

Reply:
I thought it might be that, can we updated one of the symbols to be something that will show as a type here for better illustration?

Reply:
good idea, also adding `export declare` since that is what `d.ts` files usually have when emitted by TSC